### DISCUSSION TITLE
Why do we need onnx2torch runtime dependency after the onnx model is converted to Torch

### DISCUSSION THREAD

**Discussion by satishlokkoju** [ENOT-AutoDL/onnx2torch#238] [Q&A]
I was able to successfully convert a ONNX model to Pytorch.

I observed that to run the pytorch model we need onnx2torch depndency. Why is that the case ?

Once the is converted and saved as torch model, why do we need the dependency ?

**Comment by goberreu** [comment]
Because the package uses it for testing.
